src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2081,6 +2087,48 @@ private bool RefMustBeObeyed(bool isDelegateCreation, ArgumentSyntax argumentSyn
             return hadError;
         }
 
+        private BoundExpression BindArgumentValue(DiagnosticBag diagnostics, ArgumentSyntax argumentSyntax, bool allowArglist, RefKind refKind)
+        {
+            if (argumentSyntax.Expression != null)
+            {
```

Comment:
Consider asserting that the `argumentSyntax` has no `Identifier` in this case. (either expression or out var)